dot/sync/outliers_test.go

```diff
@@ -0,0 +1,34 @@
+// Copyright 2021 ChainSafe Systems (ON)
+// SPDX-License-Identifier: LGPL-3.0-only
+
+package sync
+
+import (
+	"math/big"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_removeOutliers(t *testing.T) {
```

Comment:
a tiny bit out of scope, but can you rename that `removeOutliers` function and update its comment? It doesn't actually remove the outliers from the dataArrUint slice, it just returns some values from it.

Reply:
Renamed to `nonOutliersSumCount`